plugins/report-portal/src/components/ProjectsPage/ProjectsPage.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import {
+  Breadcrumbs,
+  Content,
+  Header,
+  HeaderIconLinkRow,
+  Page,
+  useQueryParamState,
+} from '@backstage/core-components';
+import { useRouteRef } from '@backstage/core-plugin-api';
+
+import Assessment from '@material-ui/icons/Assessment';
+
+import { rootRouteRef } from '../../routes';
+import { ProjectsPageContent } from './ProjectsPageContent/ProjectsPageContent';
+
+export const ProjectsPage = () => {
+  const rootR = useRouteRef(rootRouteRef);
```

Comment:
```suggestion
  const rootR = useRouteRef(rootRouteRef);
```

use complete/proper variables